dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -1,9 +1,8 @@
 import { ClassNames, css as emoCss } from '@emotion/react';
 
 import { border, from } from '@guardian/source-foundations';
-// @ts-ignore-start
+// @ts-ignore
 import { jsx as _jsx } from 'react/jsx-runtime';
```

Comment:
This could be simplified to a single line, as `// @ts-ignore` only targets the next line